can_tools/scrapers/official/DC/dc_vaccines.py

```diff
@@ -13,7 +14,7 @@ class DCVaccineSex(TableauDashboard):
     location_type = "state"
     baseurl = "https://dataviz1.dc.gov/t/OCTO"
     viewPath = "Vaccine_Public/Demographics"
-
+    demographic_cmu = "sex"
     data_tableau_table = "Demographics "
 
```

Comment:
adding `demographic_col_name = "Gender"` here doesn't seem to break the scraper. Is that necessary? I noticed that `demographic_col_name` exists in the other demographic scrapers

Reply:
Yes, it shouldn't hurt to add. Since the sex/gender data is returned by default (without any further filtering), I didn't use the `TableauScraper` library to extract the data, which is what the `demographic_col_name` is used for. 

I'm not sure if we're going to use this library at all, @christopherturner has some code that does filtering/selection manually (in our `TableauDashboard` class), so I think we might/will/should restructure these scrapers to use that